R/pennies_sample.R

```diff
@@ -0,0 +1,32 @@
+#' A sample of 40 pennies sampled in 2011 from the \code{pennies} data frame
+#'
+#' A dataset of 40 pennies to be treated as a sample with \code{pennies} acting
+#' as the population. Data on 
+#' these pennies were recorded in 2011.
+#'
+#' @format A data frame of 40 rows representing different pennies and 2 variables
+#' \describe{
+#'   \item{year}{Year of minting}
+#'   \item{age_in_2011}{Age in 2011}
+#' }
+#' @source StatCrunch \url{https://www.statcrunch.com/app/index.php?dataid=301596}
+#' @examples
```

Comment:
Add a `#' @seealso \code{\link{pennies}}` here so people can quickly link to original population: `pennies`